cmd/network/verification.go

```diff
@@ -0,0 +1,101 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
+	"github.com/openshift/osd-network-verifier/pkg/cloudclient"
+	"github.com/spf13/cobra"
+)
+
+var (
+	defaultTags            = map[string]string{"osd-network-verifier": "owned", "red-hat-managed": "true", "Name": "osd-network-verifier"}
+	regionEnvVarStr string = "AWS_REGION"
+	regionDefault   string = "us-east-2"
```

Comment:
```suggestion
	regionDefault   string = "us-east-1"
```

Reply:
ONV main repo uses us-east-2. Does this also need to be different? https://github.com/openshift/osd-network-verifier/blob/main/cmd/egress/cmd.go